src/resources/views/columns/closure_function.blade.php

```diff
@@ -0,0 +1,4 @@
+{{-- closure function --}}
+<span>
```

Comment:
Why not <td>?

Reply:
<td> is already added by datatable. Many other columns don't have <td></td> i.e. model_function

Reply:
You're right, my bad, the `<td>` to `<span>` rewrite was introduced with datatables-responsive plugin.